### DISCUSSION TITLE
How to remove a host from ceph cluster?

### DISCUSSION THREAD

**Discussion by microyahoo** [rook/rook#12018] [Q&A]
How to remove a host from ceph cluster after I purge all osds from this node? How to handle mgr, mon and mds on this host?

**Comment by travisn** [comment]
The OSDs and mons are the only pods with node affinity in a host-based cluster. If you drain the node, you should see all the daemons move to other nodes except the mons and osds. And if you force drain the mon, the mon should [failover to a new node](https://rook.io/docs/rook/latest/Storage-Configuration/Advanced/ceph-mon-health/#failing-over-a-monitor).

**Reply by microyahoo** [reply]
thanks, I will try it.
